src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.scss

```diff
@@ -8,34 +8,40 @@
    width: $width100;
    float: left;
    .link-header{
-        display: flex;
-        flex: 1;
+        float: left;
+        width: $width100;
         background: $color-pf-black-200;
-        flex-wrap: nowrap;
-        justify-content: space-between;
         align-items: center;
-        padding: em(5);
         .content__title{
             cursor: pointer;          
-            padding-left: em(10);
+            padding: em(22) em(5);
         }
         .selected-tab {
           font-weight: 600;
         }
-        .link-header-nav{
-            cursor: pointer;
-            flex: 1 0 auto;
-            padding: em(5) em(10);
+        .link-header-nav-list{
+            width: calc(100% - 1.5em);
             float: left;
-            border-right: 1px solid $color-pf-black-400;
-            text-align: center;
-            span{
-                float: left;
-                font-size: em(18);
-            }
-            .link-header-nav-text{
-                padding: em(5);
-                font-size: em(12);
+            padding: 0;
+            list-style: none;
+            white-space: nowrap;
+            -webkit-overflow-scrolling: touch;
+            overflow-x: scroll;
```

Comment:
Not sure how will this play out. A screenshot with overflowing content might help demonstrate the case.